packages/paste-website/src/pages/components/heading/index.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Heading
+package: '@twilio-paste/heading'
+description: Headings are words or phrases that are intended to introduce sections of content. Headings are often used for page titles, as well as subsections within the page.
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../../constants';
+import Changelog from '@twilio-paste/heading/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/heading"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/heading/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/heading"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-heading--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+### About Heading
+
+<Paragraph>Headings should allow users to easily distinguish content sections on the page.</Paragraph>
```

Comment:
why are you using the paragraph component here? I get it above as you need to place data as children.

Reply:
Copy/paste mistake.